### PR TITLE
Addition of report function

### PR DESCRIPTION
**Pull request by cfrancisco**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:

### CODE REVIEW

File: src/js/common/services/device.service.js [hunk 1]
```diff
@@ -53,5 +53,6 @@ export const getDevicesList = (page, filter) => {
 };
 
 export const getDevicesHistoryParsed = filter => {
+  console.log('getDevicesHistoryParsed', filter);
```
Comment (luisfelipefurlan) [inline src/js/common/services/device.service.js]: Tirar o console.log

File: src/js/views/dashboard/View.jsx [hunk 2]
```diff
@@ -209,26 +206,7 @@ const Dashboard = props => {
...
-            style={{ marginLeft: 10 }}
-            size='small'
-            variant='outlined'
-            color='inherit'
-            startIcon={<PauseIcon />}
-            onClick={() => stopPolling()}
-          >
-            {t('common:stop')}
-          </Button>
+          <ReportFilter t={t} />
```
Comment (luisfelipefurlan) [inline src/js/views/dashboard/View.jsx]: O ReportFilter tem que estar fora do DevelopmentContainer
Tudo que estiver dentro do DevelopmentContainer não vai aparecer na build de produção

File: src/js/views/dashboard/report/index.jsx [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+              campusDict.maxCampusPowerDemandNormalTime_value =
+                entry[key].value;
+              campusDict.maxCampusPowerDemandNormalTime_ts = entry[key].ts;
+              return;
+            }
+            // case Consumption and Surplus
+            campusDict[field] = entry[key];
+          });
+        });
+        console.log('campusDict', campusDict);
```
Comment (luisfelipefurlan) [inline src/js/views/dashboard/report/index.jsx]: Remover o console.log

File: src/js/views/dashboard/report/index.jsx [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+        // @Todo show error to user
+        console.error(error);
+      });
+  };
+
+  const getCsvCampus = finalDateIso => {
+    // 1. Do request
+    DeviceService.getDevicesHistoryParsed(queryCampusDevice('', finalDateIso))
+      .then(response => {
+        console.log('getCsvCampus', response);
```
Comment (luisfelipefurlan) [inline src/js/views/dashboard/report/index.jsx]: Remover o console.log

**Review by luisfelipefurlan — CHANGES_REQUESTED** [review]

File: src/js/views/dashboard/View.jsx (hunk 2 above)
Comment (cfrancisco) [reply src/js/views/dashboard/View.jsx]: Olha só, bom saber disso!

**Review by luisfelipefurlan — APPROVED** [review]

**Pull request merged**